x-pack/filebeat/input/internal/httplog/roundtripper.go

```diff
@@ -174,7 +180,8 @@ func logRequest(log *zap.Logger, req *http.Request, extra ...zapcore.Field) (_ *
 		errorsMessages = append(errorsMessages, fmt.Sprintf("failed to read request body: %s", err))
 	} else {
 		reqParts = append(reqParts,
-			zap.ByteString("http.request.body.content", body),
+			zap.ByteString("http.request.body.content", body[:max(0, min(len(body), maxBodyLen))]),
```

Comment:
Does this need the `max()` call? I think the purpose is to protect against negative `maxBodyLen` values, but didn't trace if those are possible.

If yes, then it seems like the same protection is needed at L114.

Reply:
This is a residue. Will remove.